static/src/stylesheets/module/content/_tweets.scss

```diff
@@ -0,0 +1,101 @@
+/* Tweets
+
+    Non-upgraded tweets are styles like the upgraded twitter embed
+    so we have font-families and colours in here that are not part
+    of the Guardian styles.
+   ========================================================================== */
+.tweet {
+    display: table-cell;
+    width: 10000px;
```

Comment:
Does this merit a comment? Or is it a CSS trick people should know (what is the trick?)?